contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -250,11 +184,72 @@ def _transcribe(item, parents, editRate, masterMobs):
     elif isinstance(item, aaf2.components.SourceClip):
         result = otio.schema.Clip()
 
-        # Evidently the last mob is the one with the timecode
-        mobs = _find_timecode_mobs(item)
-        # Get the Timecode start and length values
-        last_mob = mobs[-1] if mobs else None
-        timecode_info = _extract_timecode_info(last_mob) if last_mob else None
+        def _find_source_clip(item, slot_id=None):
+            if isinstance(item, aaf2.components.SourceClip):
+                return item
+            elif isinstance(item, aaf2.components.OperationGroup):
+                return _find_source_clip(item.segments[0])
+            elif isinstance(item, aaf2.components.Sequence):
+                return _find_source_clip(item.components[0])
+            elif isinstance(item, aaf2.components.EssenceGroup) \
+                    or isinstance(item, aaf2.components.Filler):
+                pass
```

Comment:
Is this pass because these things are not sources and you want to return `None`?

Reply:
That really should be `return None`. Still a few left overs from my process of building on the code - found an assert that I hadn't removed also.

Reply:
No problem, I figured that was it. I  just wanted to make sure it wasn’t a stubbed-out case that was waiting to be filled in.